lib/global-admin/addon/security/authentication/googleoauth/controller.js

```diff
@@ -64,6 +64,18 @@ export default Controller.extend(AuthMixin, {
     return `${ window.location.origin }`;
   }),
 
+  buttonDisabled: computed('authConfig.hostname', 'authConfig.adminEmail', 'authConfig.serviceAccountCredential', 'authConfig.oauthCredential', function() {
+    const authConfig = get(this, 'authConfig');
+    const isHostnameEmpty = isEmpty(authConfig.get('hostname'))
+    const isAdminEmailEmpty = isEmpty(authConfig.get('adminEmail'))
+
+    if (!authConfig.get('oauthCredential') || !authConfig.get('serviceAccountCredential') || isHostnameEmpty || isAdminEmailEmpty) {
```

Comment:
I'd use the module import method `get` vs the getter method. While not very likely, if `authConfig` comes back empty, null, or undefined the get method will not exist and you'll get a JS error. Using the ember `get` module method will not fail in the same manner and will be handled by ember. Stylisticlly I think its better as well since you are already using the same signature on line 68 when getting the auth config.